ci/run-integration-tests.sh

```
@@ -0,0 +1,464 @@
+    echo "services: $(adapter_services "$adapter")"
+    adapter_env_summary "$adapter"
+  done
+  exit 0
+fi
+
+preflight
+trap cleanup_all EXIT
+
+for adapter in "${selected_adapters[@]}"; do
+  compose_file="$(adapter_compose "$adapter")"
+  package="$(adapter_package "$adapter")"
```

Shellcheck (0.11.0)

[warning] package appears unused. Verify use (or export if used externally).

(SC2034)